Game/include/Scripts/ScriptList.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+#include "OrbitScript.h"
+
```

Comment:
This could probably just go in ScriptComponent as ScriptType

Reply:
In addition to having the enum, the idea for this file was to contain all the script includes in one file. That way ComponentManager doesn't need to have potentially however many Script includes in it. Just one ScriptList include that contains all the scripts.

If we're okay with that/concealing includes behind others are against style guide, the I'll move the ScriptName enum into ScriptComponent.

Reply:
Scrapped